docs/embedding/embedded-analytics-js.md

```diff
@@ -202,11 +202,11 @@ Keep in mind that API keys should only be used for testing with trusted users. A
 
 ### Set up SSO
 
-SSO is required to embed in a domain other than localhost.
+SSO is required to embed in a domain other than localhost. You can use JWT or SAML SSO. To configure SAML, see (Authenticating with SAML)[./sdk/authentication#authenticating-with-saml-sso]. To configure JWT, see the steps below.
 
-### 1. In Metabase, configure [JWT SSO](../people-and-groups/authenticating-with-jwt.md).
+#### 1. In Metabase, configure [JWT SSO](../people-and-groups/authenticating-with-jwt.md).
```

Comment:
I'm keeping these as headers because I've seen people link to them. BTW this is currently copy-pasted from SDK docs (not ideal)